pages/1.8/release-notes/index.md

```diff
@@ -395,14 +395,14 @@ For more information, see the [documentation](/services/spark/).
 - DCOS-9277 - Disabled HTTP2 in Admin Router. [enterprise type="inline" size="small" /]
 - DCOS-4298 - Time synchronization of hosts is now required. For more information, see the [system requirements](/1.8/administration/installing/ent/custom/system-requirements/#port-and-protocol).[enterprise type="inline" size="small" /]
 - DCOS-9783 - Package service broken with `java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty`. The workaround is to restart the Package service (`dcos-cosmos.service`).
-- DCOS-9804 - The DC/OS Enterprise CLI returns a spurious error message: `Failed to execute script dcos-security`.
+- DCOS-9804 - The DC/OS Enterprise CLI returns a spurious error message: `Failed to execute script dcos-security`. [enterprise type="inline" size="small" /]
 
 
 ## <a name="1-8-3"></a>1.8.3 - Sept 6, 2016
 
 ### New and changed features
 
-- You can generate custom AWS Advanced templates from the custom installer file (`dcos_generate_config.ee.sh --aws-cloudformation`) and a configuration file (`config.yaml`). Only a subset of the configuration file options are allowed (e.g. `resolvers` and `exhibitor_backend` cannot be changed). For more information, see the [documentation](/1.8/administration/installing/ent/cloud/aws/advanced/).
+- You can generate custom AWS Advanced templates from the custom installer file (`dcos_generate_config.ee.sh --aws-cloudformation`) and a configuration file (`config.yaml`). Only a subset of the configuration file options are allowed (e.g. `resolvers` and `exhibitor_backend` cannot be changed). For more information, see the [documentation](/1.8/administration/installing/ent/cloud/aws/advanced/). [enterprise type="inline" size="small" /]
```

Comment:
Are you sure that this is an enterprise feature and not just pointing to the enterprise documentation with an enterprise path? I think you might be able to do this in open source too.